packages/design-system/src/components/contextMenu/components/button.js

```diff
@@ -122,19 +126,23 @@ const Button = forwardRef(function Button(
   };
 
   return (
-    <StyledButton
-      ref={ref}
-      id={elementId}
-      tabIndex={focusedId === elementId ? 0 : -1}
-      role="menuitem"
-      isIconMenu={isIconMenu}
-      isHorizontal={isHorizontal}
-      forcePadding={forcePadding}
-      onBlur={handleBlur}
-      onClick={handleClick}
-      onFocus={handleFocus}
-      {...props}
-    />
+    <Tooltip placement={tooltipPlacement} title={title}>
+      <StyledButton
+        ref={ref}
+        id={elementId}
+        tabIndex={focusedId === elementId ? 0 : -1}
+        role="menuitem"
+        isIconMenu={isIconMenu}
+        isHorizontal={isHorizontal}
+        forcePadding={forcePadding}
+        onBlur={handleBlur}
+        onClick={handleClick}
+        onFocus={handleFocus}
+        {...props}
+      >
+        {children}
+      </StyledButton>
+    </Tooltip>
```

Comment:
💭  the right click menus don't have tooltips - so I think it'd be better to only render the tooltip when it's needed.

```js
  const Button = <StyledButton
      ref={ref}
      id={elementId}
      tabIndex={focusedId === elementId ? 0 : -1}
      role="menuitem"
      isIconMenu={isIconMenu}
      isHorizontal={isHorizontal}
      forcePadding={forcePadding}
      onBlur={handleBlur}
      onClick={handleClick}
      onFocus={handleFocus}
      {...props}
    />;

  if (title) {
    return (
       <Tooltip placement={tooltipPlacement} title={title}>
         <Button/>
       </Tooltip>
    )
  }

  return <Button />;
```

Reply:
@samwhale I've moved the `Tooltip` out of the `MenuButton` now as well per your note that the `Button` can be just wrapped into `Tooltip` as needed as we generally do instead of including it into the component.